lib/msal-core/package.json

```diff
@@ -35,6 +35,7 @@
     "atob": "~2.1.1",
     "awesome-typescript-loader": "^3.2.3",
     "babel-polyfill": "^6.26.0",
+    "chai": "*",
```

Comment:
why *? how do you avoid not getting broken for some compat issue?

Reply:
Was just trying to take most recent major version for compatibility testing. Updated in most recent commit. Also took out package-lock files